src/auth.rs

```diff
@@ -17,10 +17,16 @@ pub enum Scopes {
 /// REST API authorization.
 #[derive(Clone, Debug, PartialEq)]
 pub struct Authorization {
-    /// Authenticated subject of the request
+    /// Subject for which authorization is granted
+    /// (i.e., what may be accessed.)
     pub subject: String,
-    /// Authorization scopes of the authenticated subject
+    /// Scopes for which authorization is granted
+    /// (i.e., what types of access are permitted).
     pub scopes: Scopes,
+    /// Authenticated identity of the party to which
+    /// authorization has been granted, if available
+    /// (i.e., who is doing the accessing).
```

Comment:
Is "who is doing the accessing" actually correct here (or, at least, is it over simplifying)?  If I make a request to microservice X to access microservice Y to retrieve data from store Z, you could say that X, Y and Z are all doing the accessing, but I think the issuer would still be me.  Correct?  Maybe it's better to either drop the simplified explanation, or flesh it out?

Reply:
I've fleshed it out a bit, with reference to RFC6749 and RFC7519. Hopefully this is better?